.github/workflows/python-app.yml

```diff
@@ -26,14 +26,14 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install flake8 pytest
+        pip install flake8 pytest pytest-cov
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
     - name: Lint with flake8
       run: |
         # stop the build if there are Python syntax errors or undefined names
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
-    - name: Test with pytest
+    - name: Test and coverage with pytest and pytest-cov
       run: |
-        pytest
+        pytest --cov-config=.coveragerc --cov=audiobook_gen tests/
```

Comment:
Be very careful changing this file. Your CI may not tell you initially if something fails here if it runs from the old script, so you may not even find out it's failing until after this file is in master.

Reply:
Is this comment directed towards what I've added here, or a general comment for future updates?